### PR TITLE
#814 implemented tests for ListEnvelope

### PR DESCRIPTION
**Pull request by krzyk**
#814 
- implemented tests for covering the immutability of `listIterator()` method
- added todo to do the implementation

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#845](https://github.com/yegor256/cactoos/issues/845) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#845](https://github.com/yegor256/cactoos/issues/845) is assigned to @proshin-roman[/z](https://www.0crat.com/u/proshin-roman), here is [why](https://www.0crat.com/footprint/C63314D6Z/1290938925); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @llorllale[/z](https://www.0crat.com/u/llorllale) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

**Comment by proshin-roman** [comment]
@krzyk Could you rebase your feature branch to the latest master commit? It should fix the issue with missing test coverage report (see https://github.com/yegor256/cactoos/pull/842 for more details - it has the same issue).

**Comment by krzyk** [comment]
@proshin-roman done

**Comment by proshin-roman** [comment]
@rultor merge

File: src/test/java/org/cactoos/list/ListEnvelopeTest.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import java.util.LinkedList;
+import java.util.List;
+import java.util.ListIterator;
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * Test case for {@link ListEnvelope}.
+ *
+ * @since 0.31
```
Comment (llorllale) [inline src/test/java/org/cactoos/list/ListEnvelopeTest.java]: @krzyk we're on `0.32` now

Comment (krzyk) [reply src/test/java/org/cactoos/list/ListEnvelopeTest.java]: @llorllale fixed

**Comment by llorllale** [comment]
@rultor merge

**Comment by 0crat** [comment]
@elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova) please review this job completed by @proshin-roman[/z](https://www.0crat.com/u/proshin-roman), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#845](https://github.com/yegor256/cactoos/issues/845) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by elenavolokhova** [comment]
@proshin-roman According to our Policy:
> The code reviewer found at least three problems in the code

As far as I can see, only one issue was found here.
Please, confirm that you'll try to find at least three problems in the code in future.

**Comment by elenavolokhova** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @proshin-roman[/z](https://www.0crat.com/u/proshin-roman)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @elenavolokhova[/z](https://www.0crat.com/u/elenavolokhova)

**Pull request merged**